tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqliteTests.cs

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#if !NETCOREAPP2_1
```

Comment:
We'll still be building all the samples if we just make this change. We should update the package versions json so we don't build all the samples etc

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/3981/commits/e4f8165b811b96d984fb275c31471320439cdd38